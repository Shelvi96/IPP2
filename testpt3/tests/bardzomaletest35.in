NEW ID

1;5:507>0030826773
65585>331:3
;3>9
195>9065
40:77>40092
1354>9
36783495>3
;0397>50:6905
7;;10:86:>2
67;>9:10;9
418:2>8977817
9546;;0:2>847038
71;:3460;5>44761
012438>308434
3565>:312
2:414548:>027258:1
38607;1567>69:::6863:
@019:90;030777:945483566302
@7073580750:97:43974248
@75810
@71808;9434186;79726
@60:
@:
@3:73;75
@7124:50;5625
@27:86:74118;0;27
@12:3:01:;
@56388
@:411;56459590455;71976
@;455055781811
