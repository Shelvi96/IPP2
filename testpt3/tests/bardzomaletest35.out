321052357734912
1162327077
0
3000211
0
0
0
0
729
0
0
387480267
0
